js/src/personal/maliha/Maliha.module.css

```diff
@@ -1,72 +1,131 @@
 .title {
-  color: #b87171;
-  position: relative;
-  font-size: 125px;
-  font-family: "Playfair Display", Serif;
+  position: absolute;
+  font-size: 450px;
+  font-family: "Yeseva One", Serif;
+  letter-spacing: 6px;
+  top: 200px;
+  left: 350px;
+  font-weight: 999;
+}
+
+#introduction {
```

Comment:
Stick with classnames instead of ids for attaching styles. Classnames are hashed so they're local, but IDs are not, so if another page has #introduction as an id, you'll run into issues.